runme.py

```diff
@@ -23,8 +23,8 @@ def scenario_stock_prices(data):
 
 def scenario_co2_limit(data):
     # change global CO2 limit
-    hacks = data['hacks']
-    hacks.loc['Global CO2 limit', 'Value'] *= 0.05
+    glob = data['global']
```

Comment:
Is 'global' instead of 'glob' really to long?

Reply:
No but the word 'global' calls a function in python and can thus not be set unless it is a string

Reply:
makes sense :)